### ISSUE TITLE
qemu monitor api failed to call command like `info block`

### ISSUE THREAD

**Issue by pandada8** [proxmoxer/proxmoxer#161]
When executing the following code:
```python
pve.nodes(vm['node']).qemu(vm['vmid']).monitor.post(command="info block")
```
following exception is generated
```python
proxmoxer.core.ResourceException: 400 Bad Request: Parameter verification failed. - {'command': "type check ('string') failed - got ARRAY"}
```
This particular issue appears to have been introduced with the changes made in #90. The issue seems to revolve around the code located here: https://github.com/proxmoxer/proxmoxer/blob/d7419ab03628aa7cbcf24e90f981446996b91100/proxmoxer/backends/https.py#L197-L201
The logic in these lines of code should only activate when the url ends with `agent/exec`.

**Comment by jhollowe** [comment]
What is your PVE version and proxmoxer version. It looks like there was a change from PVE 7 to PVE 8 that changed/broke the qemu agent APIs.

**Comment by tomjones1977** [comment]
I know the fix has been merged into develop, how/when does the fix end up in the main pypi repositories?

**Comment by jhollowe** [comment]
I just released 2.1.0 which includes this fix
